geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/ingest/AbstractIngest.scala

```diff
@@ -107,16 +119,14 @@ abstract class AbstractIngest(val dsParams: Map[String, String],
           // only create the feature writer after the converter runs
           // so that we can create the schema based off the input file
           var fw: FeatureWriter[SimpleFeatureType, SimpleFeature] = null
-          val converter = createLocalConverter(file, failed)
-          // count the raw bytes read from the file, as that's what we based our total on
-          val countingStream = new CountingInputStream(new FileInputStream(file))
-          val is = PathUtils.handleCompression(countingStream, file.getPath)
+
+          val (sft, features, countingStream, closer) = getTypeAndFeatures(file, failed)
```

Comment:
so technically you moved the opening of the input stream outside of the try {} finally {} block need to move it back in